### ISSUE TITLE
Make better about page

### ISSUE THREAD

**Issue by saikat** [BrandNewCongress/website#40]

**Comment by trillian-42** [comment]
I would like to work on this.  Do I just message here to sign up?

**Comment by saikat** [comment]
Yup :). Feel free to fork the project and start working on it. Will mark
that you are on this one.

**Comment by trillian-42** [comment]
I am new to working like this, so I apologize if I have made a mess of things, but I believe I have commited changes for the page to make the grid.
